server/views/partials/analytics/layout.njk

```diff
@@ -2,6 +2,15 @@
 {% from "moj/components/sub-navigation/macro.njk" import mojSubNavigation %}
 
 {% block content %}
+  {% if prisonGroup %}
+    <div class="govuk-!-margin-bottom-4">
+      <h1 class="govuk-heading-xl govuk-!-margin-top-0 govuk-!-margin-bottom-0">
```

Comment:
Discussed with a11y folks (who overheard @ushkarev and me discussing it) whether multiple `h1` tags on a page is bad practice. It was decided that because this header is above the navigation tabs it was acceptable

Other option was to have this as `h2` which follows [Example 2: Main heading after navigation
](https://www.w3.org/WAI/tutorials/page-structure/headings), but that was potentially confusing to have `h2, h1, h2, etc`